modules/components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SocialUtil.java

```diff
@@ -47,6 +47,12 @@ public static String getTenantDomain() {
 
 	}
 
+	/**
+	 * Return the tenantDomain for the given tenantId.
+	 * @param tenantId
+	 * @return tenantDomain for the given tenantId.
```

Comment:
Need to say in the comment that "null" will be returned in some (** say which **)  error conditions, since this method returns null in some cases.